python/ray/train/_internal/checkpoint.py

```diff
@@ -96,15 +96,11 @@ def _load_checkpoint(
             # Load checkpoint from path.
             return load_checkpoint_from_path(checkpoint_to_load)
 
-    def _process_checkpoint(
+    def _process_one_checkpoint(
         self,
-        checkpoint_results: List[TrainingResult],
+        checkpoint_result: TrainingResult,
         decode_checkpoint_fn: Callable,
     ) -> None:
```

Comment:
typehint is wrong now

Reply:
seems correct to me?

Reply:
Oh oops, I mean the return typehint. I think this method returns something now.